test/unit/wicked_pdf_test.rb

```diff
@@ -1,5 +1,5 @@
 require 'test_helper'
-WickedPdf.config = { :exe_path => ENV['WKHTMLTOPDF_BIN'] || '/usr/local/bin/wkhtmltopdf' }
+WickedPdf.config = { :exe_path => ENV['WKHTMLTOPDF_BIN'] }
```

Comment:
Is there a compelling reason to remove this default? It's what I use.

Reply:
having a default at all breaks path resolution for non-standard locations. there should be zero need for a hard coded value if the binary is actually installed properly, and if they need a specific location, THEN they can provide an env variable

Reply:
^ At least that's the only thing I can vaguely remember at this point lol I've been using this patch in my own codebase since this pull request, and i can't imagine I'd ever remove a useful default unless it was breaking something else.